Packs/QualysFIM/Integrations/QualysFIM/QualysFIM_test.py

```diff
@@ -1,31 +1,51 @@
 import json
-import io
+from urllib.parse import urljoin
+import pytest
+from QualysFIM import Client
 
 BASE_URL = 'https://gateway.qg2.apps.qualys.eu/'
-DATE_FORMAT = '%Y-%m-%dT%H:%M:%S.%fZ'
+USERNAME = 'Foo'
+PASSWORD = 'Bar'
 
 
-def load_mock_response(file_name: str) -> dict:
-    """
-    Load one of the mock responses to be used for assertion.
-    Args:
-        file_name (str): Name of the mock response JSON file to return.
+def util_load_json(path: str) -> dict:
+    with open(path, encoding='utf-8') as file:
+        return json.loads(file.read())
+
+
+@pytest.fixture
+def qualysfim_client(requests_mock) -> Client:
```

Comment:
```suggestion
def authenticated_client(requests_mock) -> Client:
```

Reply:
Fixed in commit 17a3773a9323138ef3a60ac98b2740e05b1e68d1